middleware/auth.js

```diff
@@ -101,7 +108,15 @@ module.exports = function (params) {
         throw createError.BadRequest(err.message);
       }
 
-      req.session.openidTokens = tokenSet;
+      req.openIdTokens = tokenSet;
+
+      if (config.sessionSecret) {
+        let identityClaims = tokenSet.claims();
+        // Remove validation claims to reduce stored size.
+        ['aud', 'iss', 'exp', 'nonce', 'azp', 'auth_time'].forEach(claim => delete identityClaims[claim]);
```

Comment:
Would c_hash/at_hash also be included?

Reply:
@gkwang - When are those properties present?